pkgs/desktops/gnome-3/core/gnome-settings-daemon/default.nix

```diff
@@ -32,6 +32,10 @@ stdenv.mkDerivation rec {
     "-Dudev_dir=${placeholder "out"}/lib/udev"
   ];
 
+  preFixup = ''
+    gappsWrapperArgs+=(--set GSD_BACKLIGHT_HELPER "/run/wrappers/bin/gsd-backlight-helper")
```

Comment:
Maybe we could parametrize the package for this and then add something like:

```nix
    nixpkgs.overlays = [
      (self: super: {
        gnome3 = gnome3.override (gself: gsuper: {
          gnome-settings-daemon = gnome-settings-daemon.override { inherit (config.security) wrapperDir; };
        });
      })
    ];
```

It should be no-op for default `wrapperDir`.

Reply:
Sorry I don't follow.